pom.xml

```diff
@@ -189,17 +194,12 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>2.19.1</version>
+                    <version>2.22.0</version>
                     <dependencies>
                         <dependency>
                             <groupId>org.junit.platform</groupId>
```

Comment:
You should be able to get rid of junit-platform-surefire-provider. https://github.com/junit-team/junit5/issues/1536

Reply:
Removed ```junit-platform-surefire-provider``` dependency from ```maven-surefire-plugin```.